src/main.rs

```diff
@@ -417,3 +444,37 @@ fn verification_options(
 
     Ok((key_files, annotations))
 }
+
+fn sigstore_options(matches: &ArgMatches) -> Result<Option<SigstoreOpts>> {
+    let fulcio_cert = if let Some(fulcio_cert_path) = matches.value_of("fulcio-cert-path") {
+        Some(fs::read(fulcio_cert_path)?)
+    } else if Path::exists(&cli::SIGSTORE_FULCIO_CERT_PATH) {
+        Some(fs::read(&*cli::SIGSTORE_FULCIO_CERT_PATH)?)
+    } else {
+        None
+    };
+
+    let rekor_public_key =
+        if let Some(rekor_public_key_path) = matches.value_of("rekor-public-key-path") {
+            Some(fs::read(rekor_public_key_path)?)
+        } else if Path::exists(&cli::SIGSTORE_REKOR_PUBLIC_KEY_PATH) {
+            Some(fs::read(&*cli::SIGSTORE_REKOR_PUBLIC_KEY_PATH)?)
+        } else {
+            None
+        };
+
+    if fulcio_cert.is_none() && rekor_public_key.is_none() {
+        return Ok(None);
+    }
+
+    if fulcio_cert.is_none() || rekor_public_key.is_none() {
+        return Err(anyhow!(
+            "both a fulcio certificate and a rekor public key are required"
```

Comment:
I would expand that to give a hint about how to fix this problem, something like:

```suggestion
            "both a fulcio certificate and a rekor public key are required, these can be obtained by using the `cosign initialize` command"
```